File: 이민호/정렬/H-Index.js
function solution(citations) {
    var answer = 0;
    citations.sort((a,b) => (b-a));
    console.log(citations);
    for(let i = 0; i < citations.length; i++)
    {
        if (answer < Math.min(citations[i],i+1))
        {
            answer = Math.min(citations[i],i+1);
        }
    }

    return answer;
}

File: 김민기/programmers/H-Index.py
def solution(citiations):
    answer = 0
    citiations.sort()
    for i in range(len(citiations)):
        h_index = len(citiations) - i
        if citiations[i] >= h_index:
            answer = h_index
            break
    return answer

print(solution([3, 0, 6, 1, 5]))